<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chickencoop</class>
 <widget class="QDialog" name="Chickencoop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="up_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="down_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
